src/components/MainView.js

```diff
@@ -7,11 +7,30 @@ import InformationView from './InformationView';
 
 
 export default class MainView extends Component {
-	// Create the search box and link it to the UI element.
+	constructor(props) {
+		super(props);
+		this.handleMouseOver = this.handleMouseOver.bind(this);
+		this.handleMouseLeave = this.handleMouseLeave.bind(this);
+	}
+
 	componentDidMount() {
-		var input = document.getElementById('pac-input');
-		var searchBox = new google.maps.places.SearchBox(input);
+		const input = document.getElementById('pac-input');
+		const searchBox = new google.maps.places.SearchBox(input);
+	}
+
+	// add styling to searchbar //
+	handleMouseOver(el) {
+		const input = el.target;
+		const button = input.nextElementSibling.children[0];
+		button.className += ' btn-primary';
+	}
+
+	handleMouseLeave(el) {
+		const input = el.target;
+		const button = input.nextElementSibling.children[0];
+		button.classList.remove('btn-primary');
 	}
```

Comment:
I didn't use jquery here because it didnt seem necessary. It was a simple feature so I used vanilla js. But, if needed we can user jquery here